### DISCUSSION TITLE
Error: Get config Error: Command failed with exit code 3221225781

### DISCUSSION THREAD

**Discussion by dibranhoxha** [prisma/prisma#5595] [Q&A] [answered]
Hello!

When running "**prisma -v**" (version 2.16.1) on my cmd i get this error: 
Environment variables loaded from .env
Environment variables loaded from prisma\.env
Prisma schema loaded from prisma\schema.prisma
_Error: Get config Error: Command failed with exit code 3221225781: D:\Sites\xxx\server\node_modules\\@prisma\cli\query-engine-windows.exe cli get-config --ignoreEnvVarErrors 
error Command failed with exit code 1._ 

**Update:** 
I also tried reinstalling prisma as a global package, still having the same issue.
When running "**prisma -v**" (version 2.16.1) on my cmd i get this error: 
_Error: Command failed with exit code 3221225781: C:\Users\xxx-003\AppData\Local\Yarn\Data\global\node_modules\\@prisma\engines\introspection-engine-windows.exe --version_

I think this is something wrong on my WINDOWS machine cause my friend tried in MacOS and worked well.

Does anyone knows how to solve this issue?

**System Information:**
OS Name:                   Microsoft Windows 10 Pro
OS Version:                10.0.18363 N/A Build 18363
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
Original Install Date:     1/17/2021, 4:05:55 PM
System Boot Time:          11/10/2020, 7:28:01 AM
System Manufacturer:       LENOVO
System Model:              20A8S05K00
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: Intel64 Family 6 Model 69 Stepping 1 GenuineIntel ~1895 Mhz
BIOS Version:              LENOVO GRET45WW (1.22 ), 8/31/2015
Windows Directory:         C:\WINDOWS
System Directory:          C:\WINDOWS\system32
Boot Device:               \Device\HarddiskVolume2
System Locale:             en-us;English (United States)
Input Locale:              en-us;English (United States)
Time Zone:                 (UTC+01:00) Belgrade, Bratislava, Budapest, Ljubljana, Prague
Total Physical Memory:     8,072 MB
Available Physical Memory: 1,764 MB
Virtual Memory: Max Size:  15,080 MB
Virtual Memory: Available: 4,796 MB
Virtual Memory: In Use:    10,284 MB

**Node Environment:**
node version: v14.15.4
npm version: 6.14.10
yarn version: 1.22.5
prisma version: 2.16.1

**Comment by Jolg42** [comment, marked as answer]
Posted as an issue https://github.com/prisma/prisma/issues/5597

**Reply by Jolg42** [reply]
> Thanks @Jolg42 and @janpio i solved this issue after removing _@prisma/cli_ and reinstalling it and i have also installed windows build tools using _npm install --global windows-build-tools_ and restarting my machine. So i think you can close this issue.

**Reply by janpio** [reply]
Actual solution probably this: https://github.com/prisma/prisma/issues/6508#issuecomment-919483693

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
